# Awesome Genomic AI Papers ðŸ¦  + ðŸ¤– + ðŸ“„

A curated list of popular scientific papers in relation to omics. Some of topics could be genome, spatial transcriptomics and more, where the main approaches are based on machine learning (AI) techniques.

## Papers

### 2024
- **Innovative super-resolution in spatial transcriptomics: a transformer model exploiting histology images and spatial gene expression.** Chongyue Zhao, Zhongli Xu, Xinjun Wang, Shiyue Tao, William A MacDonald, Kun He, Amanda C Poholek, Kong Chen, Heng Huang, Wei Chen. **Briefing in Bioinformatics**. [[Link]](https://academic.oup.com/bib/article/25/2/bbae052/7618072).

- **CellPLM: Pre-training of Cell Language Model Beyond Single Cells.** Hongzhi Wen, Wenzhuo Tang, Xinnan Dai, Jiayuan Ding, Wei Jin, Yuying Xie, Jiliang Tang. **ICLR**. [[Link]](https://openreview.net/forum?id=BKXvPDekud)

- **DNABERT-2: Efficient Foundation Model and Benchmark For Multi-Species Genome.** Brendan F. Miller, Feiyang Huang, Lyla Atta, Arpan Sahoo & Jean Fan. **ICLR**. [[Link]](https://arxiv.org/abs/2306.15006)

- **Reference-free cell type deconvo- lution of multi-cellular pixel-resolution spatially resolved tran- scriptomics data** Zhihan Zhou, Yanrong Ji, Weijian Li, Pratik Dutta, Ramana Davuluri, Han Liu. **Nature communications**. [[Link]](https://www.nature.com/articles/s41467-022-30033-z)

- **Genegpt: Augmenting large language models with domain tools for improved access to biomedical information** Q Jin, Y Yang, Q Chen, Z Lu . **Bioinformatics**. [[Link]](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10153281/)

- **THItoGene: a deep learning method for predicting spatial transcriptomics from histological images** Yuran Jia, Junliang Liu, Li Chen, Tianyi Zhao, Yadong Wang. **Briefing in Bioinformatics**. [[Link]](https://academic.oup.com/bib/article/25/1/bbad464/7494746)

- **Spatially resolved gene expression prediction from h&e histology images via bi-modal contrastive learning.** Ronald Xie, Kuan Pang, Sai W. Chung, Catia T. Perciani, Sonya A. MacParland, Bo Wang, Gary D. Bader. **Advances in Neural Information Processing Systems**. [[Link]](https://arxiv.org/abs/2306.01859)

- **Cell clustering for spatial transcriptomics data with graph neural network** Jiachen Li, Siheng Chen, Xiaoyong Pan, Ye Yuan & Hong-Bin Shen . **Nature communications**. [[Link]](https://www.nature.com/articles/s43588-022-00266-5)

- **Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology** Daiwei Zhang, Amelia Schroeder, Hanying Yan, Haochen Yang, Jian Hu, Michelle Y. Y. Lee, Kyung S. Cho, Katalin Susztak, George X. Xu, Michael D. Feldman, Edward B. Lee, Emma E. Furth, Linghua Wang & Mingyao Li . **Nature Biotechnology**. [[Link]](https://www.nature.com/articles/s41587-023-02019-9)

- **Transformer with convolution and graph-node co-embedding: an accurate and interpretable vision backbone for predicting gene expressions from local histopatho- logical image** Xiao Xiao, Yan Kong, Ronghan Li, Zuoheng Wang, Hui Lu. **Medical Image Analysis**. [[Link]](https://www.sciencedirect.com/science/article/pii/S1361841523003006)

### 2023
- **HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution.** Eric Nguyen, Michael Poli, Marjan Faizi, Armin Thomas, Callum Birch-Sykes, Michael Wornow, Aman Patel, Clayton Rabideau, Stefano Massaroli, Yoshua Bengio, Stefano Ermon, Stephen A. Baccus, Chris RÃ©. **NeurIPS (Spotlight)**. [[Link]](https://arxiv.org/abs/2306.15794)

- **Deeptrasynergy: drug com- binations using multimodal deep learning with transformers.** Fatemeh Rafiei, Hojjat Zeraati, Karim Abbasi, Jahan B Ghasemi, Mahboubeh Parsaeian, Ali Masoudi-Nejad. **Bioinformatics**. [[Link]](https://academic.oup.com/bioinformatics/article/39/8/btad438/7226508)

- **xTrimoABFold: Improving Antibody Structure Prediction without Multiple Sequence Alignments** Yining Wang, Xumeng Gong, Shaochuan Li, Bing Yang, YiWu Sun, Chuan Shi, Hui Li, Yangang Wang, Cheng Yang, Le Song. **ICLR**. [[Link]](https://openreview.net/forum?id=F5Cj26wfiu&referrer=%5Bthe%20profile%20of%20Hui%20Li%5D(%2Fprofile%3Fid%3D~Hui_Li2))

- **Robust mapping of spatiotemporal trajectories and cellâ€“cell interactions in healthy and diseased tissues** Duy Pham, Xiao Tan, Brad Balderson, Jun Xu, Laura F. Grice, Sohye Yoon, Emily F. Willis, Minh Tran, Pui Yeng Lam, Arti Raghubar, Priyakshi Kalita-de Croft, Sunil Lakhani, Jana Vukovic, Marc J. Ruitenberg & Quan H. Nguyen . **Nature communications**. [[Link]](https://www.nature.com/articles/s41467-023-43120-6)

- **Transfer learning enables predictions in network biology** Christina V. Theodoris, Ling Xiao, Anant Chopra, Mark D. Chaffin, Zeina R. Al Sayed, Matthew C. Hill, Helene Mantineo, Elizabeth M. Brydon, Zexian Zeng, X. Shirley Liu & Patrick T. Ellinor . **Nature**. [[Link]](https://www.nature.com/articles/s41586-023-06139-9)

- **Generation of synthetic whole-slide image tiles of tumours from RNA-sequencing data via cascaded diffusion models** Francisco Carrillo-Perez, Marija Pizurica, Yuanning Zheng, Tarak Nath Nandi, Ravi Madduri, Jeanne Shen & Olivier Gevaert  **Nature Biomedical Engineering**. [[Link]](https://www.nature.com/articles/s41551-024-01193-8)

- **The Nucleotide Transformer: Building and evaluating robust foundation models for human genomics.** Hugo Dalla-Torre, Liam Gonzalez, Javier Mendoza Revilla, Nicolas Lopez Carranza, Adam Henryk Grzywaczewski, Francesco Oteri, Christian Dallago, Evan Trop, Hassan Sirelkhatim, Guillaume Richard, Marcin Skwark, Karim Beguir, Marie Lopez, Thomas Pierrot. **BioRxiv**. [[Link]](https://www.biorxiv.org/content/10.1101/2023.01.11.523679v1)


### 2022

- **Spatially aware dimension reduction for spatial transcriptomics.** Chang Xu, Xiyun Jin, Songren Wei, Pingping Wang, Meng Luo, Zhaochun Xu, Wenyi Yang, Yideng Cai, Lixing Xiao, Xiaoyu Lin, Hongxin Liu, Rui Cheng, Fenglan Pang, Rui Chen, Xi Su, Ying Hu, Guohua Wang, Qinghua Jiang. **Nucleic Acids Res**. [[Link]](https://www.nature.com/articles/s41467-022-34879-1)

- **Deepst: identifying spatial domains in spatial transcriptomics by deep learning** Shang L, Zhou X. **Nature Communications**. [[Link]](https://www.nature.com/articles/s41467-022-34879-1)

- **Genomics enters the deep learning era** Etienne Routhier1 and Julien Mozziconacci. **PeerJ**. [[Link]]()

- **Deciphering spatial domains from spatially resolved transcriptomics with an adaptive graph attention auto- encoder** Dong K, Zhang S. **Nature Communications**. [[Link]](https://www.nature.com/articles/s41467-022-29439-6)

- **Interpretable RNA Foundation Model from Unannotated Data for Highly Accurate RNA Structure and Function Predictions** Jiayang Chen, Zhihang Hu, Siqi Sun, Qingxiong Tan, Yixuan Wang, Qinze Yu, Licheng Zong, Liang Hong, Jin Xiao, Tao Shen, Irwin King, Yu Li. **BioRxiv**. [[Link]](https://www.biorxiv.org/content/10.1101/2022.08.06.503062v2.abstract)

- **Exemplar Guided Deep Neural Network for Spatial Transcriptomics Analysis of Gene Expression Prediction** Yan Yang, Md Zakir Hossain, Eric A Stone, Shafin Rahman. **WACV**. [[Link]](https://arxiv.org/abs/2210.16721)

### 2021


- **DNABERT: pre-trained Bidirectional Encoder Representations from Transformers model for DNA-language in genome.** Yanrong Ji, Zhihan Zhou, Han Liu, Ramana V Davuluri. **Bioinformatics**. [[Link]](https://academic.oup.com/bioinformatics/article/37/15/2112/6128680)

- **SPOTlight: seeded NMF regression to deconvolute spatial transcriptomics spots with single-cell transcriptomes.** Marc Elosua-Bayes, Paula Nieto, Elisabetta Mereu, Ivo Gut, Holger Heyn. **Nuclei Acid Res**. [[Link]](https://pubmed.ncbi.nlm.nih.gov/33544846/)

- **Spatial deconvolution of HER2-positive breast cancer delineates tumor-associated cell type interactions** Alma Andersson, Ludvig Larsson, Linnea Stenbeck, Fredrik SalmÃ©n, Anna Ehinger, Sunny Z Wu, Ghamdan Al-Eryani, Daniel Roden, Alex Swarbrick, Ã…ke Borg, Jonas FrisÃ©n, Camilla Engblom, Joakim Lundeberg . **Nature communications**. [[Link]](https://pubmed.ncbi.nlm.nih.gov/34650042/)

- **Spagcn: integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network.** Jian Hu, Xiangjie Li, Kyle Coleman, Amelia Schroeder, Nan Ma, David J Irwin, Edward B Lee, Russell T Shinohara, Mingyao Li. **Nature**. [[Link]](https://pubmed.ncbi.nlm.nih.gov/34711970/)

- **Spatial transcriptomics at subspot resolution with bayesspace.** Edward Zhao, Matthew R. Stone, Xing Ren, Jamie Guenthoer, Kimberly S. Smythe, Thomas Pulliam, Stephen R. Williams, Cedric R. Uytingco, Sarah E. B. Taylor, Paul Nghiem, Jason H. Bielas & Raphael Gottardo. **Nature Biotechnology**. [[Link]](https://www.nature.com/articles/s41587-021-00935-2)

- **Leveraging information in spatial transcriptomics to predict super-resolution gene expression from histology images in tumors** Minxing Pang, Kenong Su,  Mingyao Li. **bioRxiv**. [[Link]](https://www.biorxiv.org/content/10.1101/2021.11.28.470212v1)

- **Highly accurate protein structure prediction with AlphaFold** John Jumper, Richard Evans, Alexander Pritzel, Tim Green, Michael Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool, Russ Bates, Augustin Å½Ã­dek, Anna Potapenko, Alex Bridgland, Clemens Meyer, Simon A. A. Kohl, Andrew J. Ballard, Andrew Cowie, Bernardino Romera-Paredes, Stanislav Nikolov, Rishub Jain, Jonas Adler, Trevor Back, Stig Petersen, David Reiman, Ellen Clancy, Michal Zielinski, Martin Steinegger, Michalina Pacholska, Tamas Berghammer, Sebastian Bodenstein, David et al. **Nature**. [[Link]](https://www.nature.com/articles/s41586-021-03819-2)

### 2020 or older

- **Integrating spatial gene expression and breast tumour morphology via deep learning..** Bryan He, Ludvig BergenstrÃ¥hle, Linnea Stenbeck, Abubakar Abid, Alma Andersson, Ã…ke Borg, Jonas Maaskola, Joakim Lundeberg, James Zou. **Nature Biomedical Engineering**. [[Link]](https://www.nature.com/articles/s41551-020-0578-x)

- **Deep learning: new computational modelling techniques for genomics** GÃ¶kcen Eraslan, Å½iga Avsec, Julien Gagneur & Fabian J. Theis . **Naure**. [[Link]](https://www.nature.com/articles/s41576-019-0122-6)

- **A primer on deep learning in genomics** James Zou, Mikael Huss, Abubakar Abid, Pejman Mohammadi, Ali Torkamani & Amalio Telenti. **Nature genetics**. [[Link]](https://www.nature.com/articles/s41588-018-0295-5)

- **Approximate Bayesian computation with deep learning supports a third archaic introgression in Asia and Oceania** Mayukh Mondal, Jaume Bertranpetit & Oscar Lao. **Nature communications**. [[Link]](https://www.nature.com/articles/s41467-018-08089-7)

- **Machine learning applications in genetics and genomics** Maxwell W. Libbrecht & William Stafford Noble . **Nature genetics**. [[Link]](https://www.nature.com/articles/nrg3920)

## Contributing

Please feel free to [pull requests](https://github.com/Aavache/awesome-genetic-ai-papers/pulls) to add papers with reasonable impact in the domain.


## License

[![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)

To the extent possible under law, [Aavache](https://github.com/Aavache) has waived all copyright and related or neighboring rights to this work.
